spring-cloud-dataflow-server/src/test/java/org/springframework/cloud/dataflow/server/db/migration/AbstractSmokeTest.java

```diff
@@ -164,6 +164,12 @@ void shouldListJobExecutionsUsingPerformantRowNumberQuery(
 				"as STEP_COUNT FROM AGGREGATE_JOB_INSTANCE";
 		Awaitility.waitAtMost(Duration.ofSeconds(5))
 				.untilAsserted(() -> assertThat(output).contains(expectedSqlFragment));
+
+		// Verify that paging works as well
```

Comment:
Super fortunate that our smoke tests now cover this API - we get these paging tests against Oracle for basically free.